panel/src/store/modules/user.js

```diff
@@ -28,6 +28,12 @@ export default {
     current(context, user) {
       context.commit("SET_CURRENT", user);
     },
+    email(context, email) {
+      context.commit("SET_CURRENT", {
+        email: email,
+        ...context.state.current
```

Comment:
Just reading, not testing... but don't these two lines need to be switched? I'd assume there is a `email` key in `context.state.current` as well, so it would overwrite line 33 directly. Or?

Reply:
You are totally right. I just fixed it.